sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/AdvancedDiffLabeledUnlabeledData.java

```diff
@@ -62,7 +63,7 @@ public static void main(String[] args) throws IOException {
                 new RecognizeOptions()
                     .setContentType(FormContentType.APPLICATION_PDF)
                     .setIncludeFieldElements(true)
-                    .setPollInterval(Duration.ofSeconds(5)))
+                    .setPollInterval(Duration.ofSeconds(5)), Context.NONE)
```

Comment:
It would be good to add some samples that show what Context is used for.

Reply:
Was thinking of holding off on this change until we have the underlying implementation for full context passing support.